### ISSUE TITLE
20210316 Github 前端趋势榜

### ISSUE THREAD

**Issue by kujian** [kujian/githubTrending#2327]
# 20210316 Github 前端趋势榜
Github 前端开发日报栏目数据来自[Github热榜](http://news.caibaojian.com.cn/)（我开发的爬虫），每日分享HTML、CSS、JavaScript、Vue等开源项目，为开发者提供动力，点击Star按钮来关注这个项目，点击Watch来收听每日的更新[Github主页](https://github.com/kujian/githubTrending)
## JavaScript
* [LeCoupa/awesome-cheatsheets](https://github.com/LeCoupa/awesome-cheatsheets) （? Awesome cheatsheets for popular programming languages, frameworks and development tools. They include everything you should know in one single file.）
* [sveltejs/kit](https://github.com/sveltejs/kit) （A monorepo for SvelteKit and friends）
* [BMSVieira/moovie.js](https://github.com/BMSVieira/moovie.js) （Movie focused HTML5 Player）
* [yangshun/tech-interview-handbook](https://github.com/yangshun/tech-interview-handbook) （? Algorithms, front end and behavioral content for rocking your coding interview）
* [sudheerj/reactjs-interview-questions](https://github.com/sudheerj/reactjs-interview-questions) （List of top 500 ReactJS Interview Questions &amp; Answers....Coding exercise questions are coming soon!!）
* [vercel/next.js](https://github.com/vercel/next.js) （The React Framework）
* [discordjs/discord.js](https://github.com/discordjs/discord.js) （A powerful JavaScript library for interacting with the Discord API）
* [iptv-org/iptv](https://github.com/iptv-org/iptv) （Collection of 8000+ publicly available IPTV channels from all over the world）
* [goabstract/Awesome-Design-Tools](https://github.com/goabstract/Awesome-Design-Tools) （The best design tools and plugins for everything &#x1f449;）
* [dani-garcia/bitwarden_rs](https://github.com/dani-garcia/bitwarden_rs) （Unofficial Bitwarden compatible server written in Rust）
* [r3curs1v3-pr0xy/vajra](https://github.com/r3curs1v3-pr0xy/vajra) （Vajra is a highly customizable target and scope based automated web hacking framework to automate boring recon tasks and same scans for multiple target during web applications penetration testing.）
* [tobspr/shapez.io](https://github.com/tobspr/shapez.io) （shapez.io is an open source base building game inspired by factorio! Available on web &amp; desktop）
* [laurent22/joplin](https://github.com/laurent22/joplin) （Joplin - a note taking and to-do application with synchronization capabilities for Windows, macOS, Linux, Android and iOS.）
* [The-Lynx-Team/OSCP](https://github.com/The-Lynx-Team/OSCP) （Our OSCP repo: from popping shells to mental health.）
* [github/opensource.guide](https://github.com/github/opensource.guide) （Community guides for open source creators）
* [Superalgos/Superalgos](https://github.com/Superalgos/Superalgos) （Crypto Trading Bots and Strategy Automation Platform: Visual Scripting Designer, Integrated Charting System, Visual Strategy Debugger, Trading Farm Deployments.）
* [MichMich/MagicMirror](https://github.com/MichMich/MagicMirror) （MagicMirror² is an open source modular smart mirror platform. With a growing list of installable modules, the MagicMirror² allows you to convert your hallway or bathroom mirror into your personal assistant.）
* [MetaMask/metamask-extension](https://github.com/MetaMask/metamask-extension) （&#x1f310; &#x1f50c; The MetaMask browser extension enables browsing Ethereum blockchain enabled websites）
* [KikiLetGo/DimReduce](https://github.com/KikiLetGo/DimReduce) （）
* [frzyc/genshin-optimizer](https://github.com/frzyc/genshin-optimizer) （An Artifact optimizer for Genshin Impact.）
* [simondevyoutube/MinecraftClone](https://github.com/simondevyoutube/MinecraftClone) （）
* [adrianhajdin/project_chat_application](https://github.com/adrianhajdin/project_chat_application) （This is a code repository for the corresponding YouTube video. In this tutorial we are going to build and deploy a real time chat application. Covered topics: React.js, Node.js, Express.js, and Socket.io.）
* [ProjectOpenSea/opensea-creatures](https://github.com/ProjectOpenSea/opensea-creatures) （Example non-fungible collectible, to demonstrate OpenSea integration）
* [gothinkster/react-redux-realworld-example-app](https://github.com/gothinkster/react-redux-realworld-example-app) （        Exemplary real world application built with React + Redux      ）

## Vue
* [lyswhut/lx-music-desktop](https://github.com/lyswhut/lx-music-desktop) （一个基于 electron 的音乐软件）
* [PanJiaChen/vue-element-admin](https://github.com/PanJiaChen/vue-element-admin) （        vue2 admin / a management system template      ）
* [hoppscotch/hoppscotch](https://github.com/hoppscotch/hoppscotch) （&#x1f47d; A free, fast and beautiful API request builder used by 80k+ developers. https://hoppscotch.io）
* [jsososo/soso-music](https://github.com/jsososo/soso-music) （electron music app）
* [lin-xin/vue-manage-system](https://github.com/lin-xin/vue-manage-system) （        The web management system solution based on Vue2 and Element-UI.      ）
* [abhisheknaiidu/awesome-github-profile-readme](https://github.com/abhisheknaiidu/awesome-github-profile-readme) （&#x1f60e; A curated list of awesome Github Profile READMEs &#x1f4dd;）
* [cuiocean/ZY-Player](https://github.com/cuiocean/ZY-Player) （&#x25b6;&#xfe0f; 跨平台桌面端视频资源播放器.简洁无广告.免费高颜值. &#x1f39e;）
* [anncwb/vue-vben-admin](https://github.com/anncwb/vue-vben-admin) （&#x2728; &#x2728; &#x2728; A vue3 style Admin based on Vite vue3.0, vite, ant design vue 2.x, typescript，Efforts to update in progress...）
* [bawangxx/xz_voice](https://github.com/bawangxx/xz_voice) （）
* [hai-27/vue-store](https://github.com/hai-27/vue-store) （基于Vue+Vue-Router+Vuex+Element-ui+axios，参考小米商城，实现的电商项目。）
* [directus/directus](https://github.com/directus/directus) （Open-Source Data Platform — Directus wraps your new or existing SQL database with a realtime GraphQL+REST API for developers, and an intuitive admin app for non-technical users. &#x1f430;）
* [bailicangdu/vue2-elm](https://github.com/bailicangdu/vue2-elm) （        基于 vue2 + vuex 构建一个具有 45 个页面的大型单页面应用      ）
* [chuzhixin/vue-admin-beautiful](https://github.com/chuzhixin/vue-admin-beautiful) （vue-admin-beautiful是一款绝佳的前端开发框架（基于vue/cli 4 最新版，同时支持电脑，手机，平板）,长期更新维护,感谢您的star,我一直在努力 Vue admin beautiful is an excellent front-end development framework (supporting computers, mobile phones and tablets at the same time), long-term update and maintenance, thank you for your star, I have been working hard）
* [beekeeper-studio/beekeeper-studio](https://github.com/beekeeper-studio/beekeeper-studio) （Modern and easy to use SQL client for MySQL, Postgres, SQLite, SQL Server, and more. Linux, MacOS, and Windows.）
* [vueComponent/ant-design-vue-pro](https://github.com/vueComponent/ant-design-vue-pro) （An enterprise-class UI components based on Ant Design and Vue. &#x1f41c;）
* [chaitin/xray](https://github.com/chaitin/xray) （一款完善的安全评估工具，支持常见 web 安全问题扫描和自定义 poc | 使用之前务必先阅读文档）
* [DataV-Team/DataV](https://github.com/DataV-Team/DataV) （Vue数据可视化组件库（类似阿里DataV，大屏数据展示），提供SVG的边框及装饰、图表、水位图、飞线图等组件，简单易用，长期更新(React版已发布)）
* [weilanwl/ColorUI](https://github.com/weilanwl/ColorUI) （鲜亮的高饱和色彩，专注视觉的小程序组件库）
* [samzx/soft-portfolio](https://github.com/samzx/soft-portfolio) （Neumorphic personal portfolio with dark mode &#x1f4a1;）
* [bailicangdu/vue2-manage](https://github.com/bailicangdu/vue2-manage) （        基于 vue + element-ui 的后台管理系统      ）
* [JakHuang/form-generator](https://github.com/JakHuang/form-generator) （&#x2728;Element UI表单设计及代码生成器）
* [salomonelli/best-resume-ever](https://github.com/salomonelli/best-resume-ever) （? ? Build fast ? and easy multiple beautiful resumes and create your best CV ever! Made with Vue and LESS.）
* [dcloudio/uni-ui](https://github.com/dcloudio/uni-ui) （基于uni-app的ui框架）
* [maomao1996/Vue-mmPlayer](https://github.com/maomao1996/Vue-mmPlayer) （基于Vue2的在线音乐播放器（PC） Online music player （vue-music））
* [frappe/books](https://github.com/frappe/books) （Free Desktop book-keeping software for small-businesses and freelancers.）
* [ElemeFE/element](https://github.com/ElemeFE/element) （        A Vue.js 2.0 UI Toolkit for Web      ）
* [Armour/vue-typescript-admin-template](https://github.com/Armour/vue-typescript-admin-template) （&#x1f596; A vue + typescript minimal admin template）
* [macrozheng/mall-admin-web](https://github.com/macrozheng/mall-admin-web) （mall-admin-web是一个电商后台管理系统的前端项目，基于Vue+Element实现。 主要包括商品管理、订单管理、会员管理、促销管理、运营管理、内容管理、统计报表、财务管理、权限管理、设置等功能。）
* [cadriel/fluidd](https://github.com/cadriel/fluidd) （Fluidd, a klipper web client.）
* [WishMelz/imgurl](https://github.com/WishMelz/imgurl) （图床/超简图床/基于github/免费）
* [vuestorefront/storefront-ui](https://github.com/vuestorefront/storefront-ui) （Customization-first, performance-oriented and elegant UI framework for eCommerce (and not only) based on Vue.js and Google Retail UX Playbook. Made with &#x1f49a; by Vue Storefront team and contributors.）
* [CareyWang/sub-web](https://github.com/CareyWang/sub-web) （）
* [recoluan/vuepress-theme-reco](https://github.com/recoluan/vuepress-theme-reco) （this is a vuepress theme）
* [elunez/eladmin-web](https://github.com/elunez/eladmin-web) （eladmin前端源码，项目基于 Spring Boot 2.1.0 、 Spring Boot Jpa、 Spring Security、Redis、Vue的前后端分离后台管理系统， 权限控制采用 RBAC，菜单动态路由）
* [xugaoyi/vuepress-theme-vdoing](https://github.com/xugaoyi/vuepress-theme-vdoing) （&#x1f680;一款简洁高效的VuePress 知识管理&amp;博客(blog) 主题）
* [bradtraversy/vue-crash-2021](https://github.com/bradtraversy/vue-crash-2021) （Task Tracker project from youtube crash course）

## HTML
* [docker/awesome-compose](https://github.com/docker/awesome-compose) （Awesome Docker Compose samples）
* [nicehash/NiceHashQuickMiner](https://github.com/nicehash/NiceHashQuickMiner) （Super simple &amp; easy Windows 10 cryptocurrency miner made by NiceHash.）
* [tc39/ecma262](https://github.com/tc39/ecma262) （        Status, process, and documents for ECMA262      ）
* [tailwindlabs/tailwindcss-forms](https://github.com/tailwindlabs/tailwindcss-forms) （）
* [DeviaVir/zenbot](https://github.com/DeviaVir/zenbot) （Zenbot is a command-line cryptocurrency trading bot using Node.js and MongoDB.）
* [vernesong/OpenClash](https://github.com/vernesong/OpenClash) （A Clash Client For OpenWrt）
* [gustavoguanabara/html-css](https://github.com/gustavoguanabara/html-css) （Curso de HTML5 e CSS3）
* [sumitkumar1503/schoolmanagement](https://github.com/sumitkumar1503/schoolmanagement) （School Management System || Python Django）
* [unblocked-pw/unblocked-pw.github.io](https://github.com/unblocked-pw/unblocked-pw.github.io) （Github mirror for Unblocked）
* [aave/aave-protocol](https://github.com/aave/aave-protocol) （Aave Protocol Version 1.0 - Decentralized Lending Pools）
* [daattali/beautiful-jekyll](https://github.com/daattali/beautiful-jekyll) （&#x2728; Build a beautiful and simple website in literally minutes. Demo at http://deanattali.com/beautiful-jekyll）
* [adityatelange/hugo-PaperMod](https://github.com/adityatelange/hugo-PaperMod) （Hugo Theme PaperMod）
* [ephtracy/ephtracy.github.io](https://github.com/ephtracy/ephtracy.github.io) （）
* [xiaorouji/openwrt-passwall](https://github.com/xiaorouji/openwrt-passwall) （）
* [Garderoben/MudBlazor.Templates](https://github.com/Garderoben/MudBlazor.Templates) （Ready to use Blazor Templates in different styles and layout with all the basic setup already done for MudBlazor.）
* [rfordatascience/tidytuesday](https://github.com/rfordatascience/tidytuesday) （Official repo for the #tidytuesday project）
* [akashyap2013/PortFolio_Website](https://github.com/akashyap2013/PortFolio_Website) （Complete Portfolio Website with Bootstrap - HTML/CSS In this project, we are going to learn and build how to create a complete portfolio website with bootstrap using HTML and CSS. We will understand everything from scratch.）
* [peng-zhihui/peng-zhihui.github.io](https://github.com/peng-zhihui/peng-zhihui.github.io) （稚晖的个人站）
* [geerlingguy/raspberry-pi-pcie-devices](https://github.com/geerlingguy/raspberry-pi-pcie-devices) （Raspberry Pi PCI Express device compatibility database）

## CSS
* [hasinhayder/tailwind-cards](https://github.com/hasinhayder/tailwind-cards) （A growing collection of text/image cards you can use/copy-paste in your tailwind css projects）
* [ryanoasis/nerd-fonts](https://github.com/ryanoasis/nerd-fonts) （? Iconic font aggregator, collection, and patcher: 39 patched fonts (87k+ variations). Includes popular glyph collections such as Font Awesome &amp; fonts such as Hack）
* [houshanren/hangzhou_house_knowledge](https://github.com/houshanren/hangzhou_house_knowledge) （2017年买房经历总结出来的买房购房知识分享给大家，希望对大家有所帮助。买房不易，且买且珍惜。Sharing the knowledge of buy an own house that according to the experience at hangzhou in 2017 to all the people. It's not easy to buy a own house, so I hope that it would be useful to everyone.）
* [animate-css/animate.css](https://github.com/animate-css/animate.css) （&#x1f37f; A cross-browser library of CSS animations. As easy to use as an easy thing.）
* [jgthms/bulma](https://github.com/jgthms/bulma) （        Modern CSS framework based on Flexbox      ）
* [bradtraversy/50projects50days](https://github.com/bradtraversy/50projects50days) （50+ mini web projects using HTML, CSS &amp; JS）
* [prakhar1989/docker-curriculum](https://github.com/prakhar1989/docker-curriculum) （? A comprehensive tutorial on getting started with Docker!）
* [hasinhayder/tailwind-boilerplate](https://github.com/hasinhayder/tailwind-boilerplate) （Tailwind CSS + Vite Boilerplate）
* [creativetimofficial/tailwind-starter-kit](https://github.com/creativetimofficial/tailwind-starter-kit) （Tailwind Starter Kit a beautiful extension for TailwindCSS, Free and Open Source）
* [v2fly/v2ray-examples](https://github.com/v2fly/v2ray-examples) （v2ray-core 的模板们）
* [NYRI4/Comfy](https://github.com/NYRI4/Comfy) （Stay comfy while talking to your friends）
* [morpheusthewhite/spicetify-themes](https://github.com/morpheusthewhite/spicetify-themes) （A community-driven collection of themes for Spicetify (https://github.com/khanhas/spicetify-cli)）
* [adobe-fonts/source-code-pro](https://github.com/adobe-fonts/source-code-pro) （Monospaced font family for user interface and coding environments）
* [hankchizljaw/modern-css-reset](https://github.com/hankchizljaw/modern-css-reset) （A bare-bones CSS reset for modern web development.）
* [vinceliuice/Orchis-theme](https://github.com/vinceliuice/Orchis-theme) （Orchis is a [Material Design](https://material.io) theme for GNOME/GTK based desktop environments.）
* [bedimcode/portfolio-responsive-complete](https://github.com/bedimcode/portfolio-responsive-complete) （）
* [ronggang/transmission-web-control](https://github.com/ronggang/transmission-web-control) （        Transmission Web Control is a custom web UI. The project began in Google Code.      ）
* [vinceliuice/Qogir-theme](https://github.com/vinceliuice/Qogir-theme) （Qogir is a flat Design theme for GTK）
* [developedbyed/glass-website](https://github.com/developedbyed/glass-website) （）
* [john-smilga/react-projects](https://github.com/john-smilga/react-projects) （）
* [PoojaMittal2842/Delhi-Tourism](https://github.com/PoojaMittal2842/Delhi-Tourism) （Website on Delhi Tourism）

## 关于我
 [前端开发博客](http://caibaojian.com.cn/) 站长，[微博](https://weibo.com/kujian) 粉丝18万，[微信公众号](https://open.weixin.qq.com/qr/code?username=caibaojian_com)。
## 推荐
 关注我的微信公众号：[前端开发博客](https://open.weixin.qq.com/qr/code?username=caibaojian_com)，在后台回复以下关键字获取资源：
 1. 回复「1024」，领取前端进阶资料，包含小程序、Nodejs、Git等
 2. 回复「Vue」获取 Vue 精选文章
 3. 回复「面试」获取 面试 精选文章
 4. 回复「JS」获取 JavaScript 精选文章
 5. 回复「CSS」获取 CSS 精选文章
 6. 回复「加群」进入500人前端精英群
 7. 回复「电子书」下载我整理的大量前端资源，含面试、Vue实战项目、CSS和JavaScript电子书等。
 8. 回复「知识点」下载高清JavaScript知识点图谱
